core/core_test.go

```diff
@@ -50,7 +50,7 @@ func TestCreate(t *testing.T) {
 
 func TestSummary(t *testing.T) {
 	createJson := []byte("{\"Name\":\"Alice\",\"Password\":\"Hello\"}")
-	delegateJson := []byte("{\"Name\":\"Bob\",\"Password\":\"Rob\",\"Time\":\"2h\",\"Uses\":1}")
+	delegateJson := []byte("{\"Name\":\"Bob\",\"Password\":\"Rob\",\"Time\":\"2h\",\"Uses\":1,\"AnyUser\":true}")
```

Comment:
Good point. I added `AnyUser: true` to preserve the original behavior of the tests, but I agree that we should test the alternate path as well.